examples/qwik/src/routes/index.tsx

```diff
@@ -0,0 +1,109 @@
+import { component$, useServerMount$, useSignal } from '@builder.io/qwik'
+import { grafbaseClient } from '~/utils/grafbase'
+
+interface Plants {
+  plantCollection: {
+    edges: Plant[]
+  }
+}
+
+interface Plant {
+  id: string
+  name: string
+  description: string
+}
+
+export const GetAllPlantsQuery = `
+  query GetAllPlants($first: Int!) {
+    plantCollection(first: $first) {
+      edges {
+        node {
+          id
+          name
+          description
+        }
+      }
+    }
+  }
+`
+
+export const AddNewPlantMutation = `
```

Comment:
I'd get rid of this for the example. We can just read from Grafbase and add to the README details on how to run the mutation using the playground.